src/test/java/service/RevenueCalculatorTest.java

```diff
@@ -0,0 +1,173 @@
+package service;
+
+import static domain.card.Value.ACE;
+import static domain.card.Value.KING;
+import static domain.card.Value.ONE;
+import static domain.card.Value.SEVEN;
+import static domain.card.Value.TWO;
+import static fixture.RevenueFixture.getPlayerRevenue;
+import static fixture.RevenueFixture.giveTwoCardToPlayer;
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+
+import domain.Dealer;
+import domain.GameResults;
+import domain.Player;
+import domain.Players;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RevenueCalculatorTest {
+
+    private final int PLAYER_BETTING_AMOUNT = 1000;
+    private final int BLACKJACK_PLAYER_BETTING_AMOUNT = 1500;
+
+    private Dealer dealer;
+    private Player player;
+    private Players players;
+    private RevenueCalculator calculator;
+
+    @BeforeEach
+    void init() {
+        dealer = Dealer.createDefaultDealer();
+        player = Player.createDefault("test", 1000);
+        players = Players.createInitialPlayers(List.of(player));
+        calculator = RevenueCalculator.createCalculator(dealer, players);
+    }
+
+    @Test
+    @DisplayName("플레이어가 블랙잭일 때, 플레이어는 배팅금의 1.5배를 얻고, 딜러는 그만큼 잃는다.")
+    void when_only_player_is_black_jack() {
+        // given
+        giveTwoCardToPlayer(ACE, KING, player);
+        giveTwoCardToPlayer(ONE, SEVEN, dealer);
+
+        // when
+        GameResults results = calculator.calculateRevenues();
+        double playerRevenue = getPlayerRevenue(results, player);
+        double dealerRevenue = getPlayerRevenue(results, dealer);
+
+        // then
+        assertSoftly((softly) -> {
+            softly.assertThat(playerRevenue).isEqualTo(BLACKJACK_PLAYER_BETTING_AMOUNT);
+            softly.assertThat(dealerRevenue).isEqualTo(-BLACKJACK_PLAYER_BETTING_AMOUNT);
+        });
```

Comment:
```suggestion
        GameResults results = calculator.calculateRevenues();
        
        // then
        double playerRevenue = getPlayerRevenue(results, player);
        double dealerRevenue = getPlayerRevenue(results, dealer);
        
        assertSoftly((softly) -> {
            softly.assertThat(playerRevenue).isEqualTo(BLACKJACK_PLAYER_BETTING_AMOUNT);
            softly.assertThat(dealerRevenue).isEqualTo(-BLACKJACK_PLAYER_BETTING_AMOUNT);
        });
```

++ `BLACKJACK_PLAYER_BETTING_AMOUNT`를 사용하지 말고 다르게 표현해보면 어떨까요?
만약에 1.5배의 금액이라는 정책에서 2배로 바꾼다면 테스트가 깨질 것 같아요.

리팩토링 할 때에도 용이하게 하기 위해서 테스트가 있는데 `FIRST` 원칙에 대해서 공부해보면 좋을 것 같습니다~